src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/ExecutionEngineCode/ExecutionEngine.cs

```diff
@@ -501,7 +502,7 @@ private void DoScriptExecution(bool isBatchParser)
             }
             else
             {
-                ExecuteBatchInternal(script, /* num */ 1, /* lineNumber */ 0);
+                ExecuteBatchInternal(script, /* num */ 1, /* lineNumber */ 0, /* sqlcmdCommand required for parsing only*/ null);
```

Comment:
Suggest using named parameters (comments like 'required for parsing only' can still be kept)